src/plugins/console/public/application/containers/main/main.tsx

```diff
@@ -51,19 +60,47 @@ export function Main({ isEmbeddable = false }: MainProps) {
         <EuiFlexItem>
           <EuiSplitPanel.Outer grow={true} borderRadius={isEmbeddable ? 'none' : 'm'}>
             <EuiSplitPanel.Inner grow={false} className="consoleTabs">
-              <TopNavMenu
-                disabled={!done}
-                items={getTopNavConfig({
-                  selectedTab,
-                  setSelectedTab,
-                })}
-              />
+              <EuiFlexGroup direction="row" alignItems="center">
+                <EuiFlexItem>
+                  <TopNavMenu
+                    disabled={!done}
+                    items={getTopNavConfig({
+                      selectedTab,
+                      setSelectedTab,
+                    })}
+                  />
+                </EuiFlexItem>
+                <EuiFlexItem grow={false}>
+                  <EuiButtonIcon iconType="save" onClick={() => {}} />
+                </EuiFlexItem>
+                <EuiFlexItem grow={false}>
+                  <EuiButtonIcon iconType="keyboard" color="primary" />
+                </EuiFlexItem>
+                <EuiFlexItem grow={false}>
+                  <EuiButtonIcon iconType="questionInCircle" color="primary" />
+                </EuiFlexItem>
+              </EuiFlexGroup>
             </EuiSplitPanel.Inner>
+            <EuiHorizontalRule margin="none" />
             <EuiSplitPanel.Inner paddingSize="none">
               {selectedTab === SHELL_TAB_ID && (
                 <Editor loading={!done} setEditorInstance={() => {}} />
               )}
             </EuiSplitPanel.Inner>
+            <EuiHorizontalRule margin="none" />
+            <EuiSplitPanel.Inner paddingSize="xs" grow={false}>
+              <EuiButtonEmpty
+                onClick={() => {}}
+                iconType="editorCodeBlock"
+                size="xs"
+                color="text"
+                css={{ fontWeight: 'normal' }}
```

Comment:
Is this necessary @julianrosado ?

Reply:
Nah, I think it's fine with the default. That is probably more my fault for not being clearer, and Elena trying to closely match my design. @ElenaStoeva totally good to use the default here.

Reply:
Thanks for clarifying. To be extra clear, we're suggesting to remove line 97.